### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**

### CODE REVIEW

File: src/Components/TopBar.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+export default function TopBar() {
+  return (
+    <section className="jumbotron text-center">
+      <h1 className="jumbotron-heading">Go Fund Mo</h1>
```
Comment (mikeboyle) [inline src/Components/TopBar.js]: Go Fund Mo? Is it Mo or Me?
(There used to be an instructor here called Mo so I don't know if this is a legacy inside joke)

Comment (JoshuaKGoldberg) [reply src/Components/TopBar.js]: Haha I assumed this was a legacy inside joke! But if he's no longer kicking around I'll revert back to Me. 😄

Comment (JoshuaKGoldberg) [reply src/Components/TopBar.js]:
```suggestion
      <h1 className="jumbotron-heading">Go Fund Me</h1>
```

**Comment by mikeboyle** [comment]
Closing and reopening for circle

**Pull request closed without merging**